lib/dockly/deb.rb

```diff
@@ -127,14 +127,14 @@ def convert_package
   end
 
   def add_foreman(package)
-    return if foreman.nil?
-    info "adding foreman export"
-    foreman.create!
-    package.attributes[:prefix] = foreman.init_dir
-    Dir.chdir(foreman.build_dir) do
-      package.input('.')
+    return if (foreman || []).empty?
+    foreman.each do |fore|
+      info "adding foreman export '#{fore.name}'"
+      fore.create!
+      package.attributes[:prefix] = fore.init_dir
+      Dir.chdir(fore.build_dir) { package.input('.') }
```

Comment:
Does this just stage a copy when we convert from a Dir package to a Deb/RPM package?  It doesn't copy immediately?